### DISCUSSION TITLE
Border issue

### DISCUSSION THREAD

**Discussion by leenorshn** [tailwindlabs/tailwindcss#8284] [Ideas]
Hi guys. 
Good job for all havy work you are doing.

I'm trying to use **border** utility in a input but when i trying to tap **border-1** it doesn't exist , i know that JIT mode can help me for that by doing something like border-[1px], but i think it can be better to have a list like this
border-0
border-1
border-2
border-3
border-4
....
as you did for padding, margin .

Thanks you for all!

**Comment by NovusOpus** [comment]
The `border` class have a 1px border as you can see in the [documentation](https://tailwindcss.com/docs/border-width) , the same applies to other classes/utilities, just writing the initial class without a number for a 1px value.

**Reply by leenorshn** [reply]
Thanks you, but what about 3px or 5px the number skipped on the list of border??? I mean border-3 or border-5

**Comment by jenstornell** [comment]
I agree about border-3. It's often used for my sites as well and I would not mind to have such a common value built in.
